src/client/components/notification.vue

```diff
@@ -13,7 +13,7 @@
 			<i v-else-if="notification.type === 'mention'" class="fas fa-at"></i>
 			<i v-else-if="notification.type === 'quote'" class="fas fa-quote-left"></i>
 			<i v-else-if="notification.type === 'pollVote'" class="fas fa-poll-h"></i>
-			<XReactionIcon v-else-if="notification.type === 'reaction'" :reaction="notification.reaction" :custom-emojis="notification.note.emojis" :no-style="true"/>
+			<XReactionIcon v-else-if="notification.type === 'reaction'" :reaction="notification.reaction ? notification.reaction.replace(/^:(\w+):$/, ':$1@.:') : notification.reaction" :custom-emojis="notification.note.emojis" :no-style="true"/>
```

Comment:
nullになることなさそうだから
```suggestion
			<XReactionIcon v-else-if="notification.type === 'reaction'" :reaction="notification.reaction.replace(/^:(\w+):$/, ':$1@.:')" :custom-emojis="notification.note.emojis" :no-style="true"/>
```
でよさそう

Reply:
> nullになることなさそうだから

まずないんだけど完全に自信がなくて、あとここでoptional chaining使うと一部ブラウザで刺さるので念の為こうしたわ。

Reply:
あんまり細かく見る時間がなかったからとりあえず安全な方にしたけど、気になるならなおしてもいいのだわ。

Reply:
コメント残した